src/Payload/Options.php

```diff
@@ -1,17 +1,23 @@
 <?php
-
+declare (strict_types=1);
 namespace Xrpl\XummSdkPhp\Payload;
 
 final class Options
 {
+    /**
+     * @param array<string>|null $signers
+     */
     public function __construct(
         public readonly ?bool $submit = null,
         public readonly ?bool $multisign = null,
         public readonly ?int $expire = null,
         public readonly ?bool $immutable = null,
         public readonly ?bool $forceAccount = null,
-        public readonly ?ReturnUrl $returnUrl = null,
         public readonly ?array $signers = null,
+        public readonly ?ReturnUrl $returnUrl = null,
     ) {
+        if($signers) $this->typeArrayNullString(...$signers);
+    }
+    private function typeArrayNullString(?string ...$arg): void {
```

Comment:
Thanks for the update. Could you please throw an exception here? Refer to the `Exception` namespace to see how other exceptions are created. Maybe something like an `InvalidOptionsException` within the `Exception\Payload` namespace.

And again a test case would be good. Ideally a unit test for the constructor asserting the exception is thrown, and to add some `signers` strings to the serialization test

Reply:
@paulinevos it throws an exception as I’m using `declare (strict_types=1);`. Thought that looks a bit neater. @Dennis-Mwea also added the unit test. Good to go?